code-challenges.test.js

```diff
@@ -15,42 +15,173 @@
+
+        //   ● sequencer › takes in a number(greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
+
+        // ReferenceError: sequencer is not defined
+
+
+
 // b) Create the function that makes the test pass.
 
+// PSEUDO CODE
+// ----------------------
+// Create a FUNCTION called SEQUENCER that takes in a NUMBER > 2
+// Create a VARIABLE to hold an ARRAY of first 2 values of fibonacci sequence
+// Per established rules of the fibonacci sequesnce, ITERATE the number of times in the paramaters argument
+// Fill the newly created variable with the fibonacci sequence via the PUSH METHOD.
+
+
+const sequencer = (number) => {
+    let array = [1, 1]
+    for (let i = 0; i < number - 2; i++) {
+        if (number > 2) {
+            array.push(array[i] + array[i + 1])
+        }
+    }
+    return array
+}
+
+        // JEST "GOOD" TEST PASS
+        // -------------------------
+            // PASS./ code - challenges.test.js
+            // sequencer
+            //     ✓ takes in a number(greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence(4 ms)
+
+
+
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 7, 9, 199]
+describe("oddNumberSorter", () => {
+    const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+    const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+    it("takes in an array and returns a new array of only odd numbers sorted from least to greatest", () => {
+        expect(oddNumberSorter(fullArr1)).toEqual([-9, 7, 9, 199])
+        expect(oddNumberSorter(fullArr2)).toEqual([-823, 7, 23])
+    })
+})
 
-const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+// Expected output: [-9, 7, 9, 199]
 // Expected output: [-823, 7, 23]
 
 
+        // JEST "GOOD" TEST FAIL
+        // -------------------------
+        // FAIL./ code - challenges.test.js
+        // oddNumberSorter
+        //     ✕ takes in an array and returns a new array of only odd numbers sorted from least to greatest(1 ms)
+
+        //   ● oddNumberSorter › takes in an array and returns a new array of only odd numbers sorted from least to greatest
+
+        // ReferenceError: oddNumberSorter is not defined
+
+
+
 // b) Create the function that makes the test pass.
 
 
+// PSEUDO CODE
+// ----------------------
+// Create a FUNCTION called oddNumberSorter that takes in an ARRAY
+// Return the supplied array and FILTER it based on TYPEOF NUMBER and whether it is ODD NUMBER
+// Return the array and SORT it least to greatest
+
+
+const oddNumberSorter = (array) => {
```

Comment:
Logic looks great on this problem!  A way to simplify this a little further is to remove the conditional.  .filter has a built-in conditional statement, so there is no need to add one in.